app/bundles/CoreBundle/Helper/PathsHelper.php

```diff
@@ -162,8 +162,10 @@ public function getSystemPath($name, $fullPath = false)
 
         if ($fullPath) {
             $rootPath = (!empty($this->paths[$name.'_root'])) ? $this->paths[$name.'_root'] : $this->paths['root'];
-
-            return $rootPath.'/'.$path;
+            
+			if(strpos($path, $rootPath) === false){
```

Comment:
The problem with Travis is that you used tabs. All Mautic files use 4 spaces instead. Please fix that indentation and the TravisCI check will be fixed.